linux/debian/libkeymancore2.symbols

```diff
@@ -8,7 +8,7 @@ libkeymancore.so.1 libkeymancore1 #MINVER#
  km_core_context_items_dispose@Base 17.0.195
  km_core_context_length@Base 17.0.195
  km_core_context_set@Base 17.0.195
- km_core_cp_dispose@Base 17.0.263
+ km_core_cu_dispose@Base 18.0.28
```

Comment:
This needs to be the current version otherwise the API check will fail. Possibly the check is not smart enough...

```suggestion
 km_core_cu_dispose@Base 18.0.30
```